tests/functional/splinter_test.c

```diff
@@ -579,13 +609,13 @@ do_operation(test_splinter_thread_params *params,
 
 /*
  *-----------------------------------------------------------------------------
- *
  * test_trunk_insert_lookup_thread
  *
- *      do number of insert_granularity inserts followed by number of
+    RESOLVE: Comments need rewording: ... Suggestions?
```

Comment:
Kindly advise .... I tried to glean what the code author meant with comment below ... but failed.

Need suggestions for rewording ...

Reply:
I'm also not really following what's going on here. Clearly this thread is doing some mixture of operations, but the exact details aren't immediately obvious to me either from the comment or the code itself.

I *think* that what's happening is that there is a global `op_granularity` of say 100. Then the thread will do say 30 inserts, 50 lookups and 20 range lookups in each 100 operation block.

Reply:
Reworded the comment to be clearer, .. somewhat.